### ISSUE TITLE
Make "Start Bash Shell" feature configurable

### ISSUE THREAD

**Issue by xiaoyinl** [microsoft/winfile#44]
The File Manager supports "Start Cmd Shell" and "Start PowerShell". Can we add "Start Bash Shell", if users have installed Bash on Windows?

Edit:
* [X] Add "Start Bash Shell" (Done in #61)
* [ ] Make this feature configurable

**Comment by Tadas** [comment]
With all the requests to include their favourite shell seems like this should be a configurable setting

**Comment by My1** [comment]
or maybe instead of doing it shell-specific just have a "start whatever you want here" as there may be other programs that can like things like this.

**Comment by NazmusLabs** [comment]
Yeah, I definitely think a configuration setting would be better because of all the different shells Windows 10 now supports, this is especially true because now you can have multiple Linux distros with indivudual launch commands, such as "ububtu" or "kali".

I think we should be able to just add or remove different shell launch options from the menu to prevent it from getting too cluttered.

For instance, I may only want powershell 6 and bash. I'd rather not clutter up the menu with cmd.

**Comment by My1** [comment]
add/remove sounds awesome.
